docs/sql-ref-syntax-qry-select-window.md

```diff
@@ -42,14 +42,24 @@ window_function OVER
 
     * Analytic Functions
 
-      **Syntax:** `CUME_DIST | LAG | LEAD`
+      **Syntax:** `CUME_DIST | LAG | LEAD | NTH_VALUE | FIRST_VALUE | LAST_VALUE`
 
     * Aggregate Functions
 
       **Syntax:** `MAX | MIN | COUNT | SUM | AVG | ...`
 
       Please refer to the [Built-in Aggregation Functions](sql-ref-functions-builtin.html#aggregate-functions) document for a complete list of Spark aggregate functions.
 
+* **nulls_option**
+
+    Specified to indicate whether skip null values in the determination of which row to use. `RESPECT NULLS` is supported by default if you do not specify `IGNORE NULLS`.
```

Comment:
Specifies whether or not to skip null values when evaluating the window function. `RESECT NULLS` means not skipping null values, while `IGNORE NULLS` means skipping. If not specified, the default is `RESECT NULLS`.